components/formats-bsd/src/loci/formats/out/TiffWriter.java

```diff
@@ -343,6 +343,9 @@ public void close() throws IOException {
     if (in != null) {
       in.close();
     }
+    if (tiffSaver != null) {
+      tiffSaver.close();
```

Comment:
If Bio-Formats is to be stuck at Java 6 for some time yet, so without `AutoCloseable`, I wonder if it would be useful in such a review to also add a,
```java
@Override
protected void finalize() {
    close();
}
```
Just an idea, though, and outwith this PR.